.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -78,3 +78,9 @@ jobs:
         with:
           name: karmada_e2e_log_${{ matrix.k8s }}
           path: ${{ github.workspace }}/karmada-e2e-logs/${{ matrix.kubeapiserver-version }}-${{ matrix.karmada-version }}/
+      - name: upload kind logs
+        if: always()
```

Comment:
How about upload the log only when failure?

Reply:
done

Reply:
> How about upload the log only when failure?

Just out of curious, why?

Reply:
Success log also provides valuable information, at least it can prove the log-collection functionality works as expected.

No concern about storage or performance by the way.

Reply:
> Success log also provides valuable information, at least it can prove the log-collection functionality works as expected. No concern about storage or performance by the way.

I think what you're saying makes more sense.